modules/test/playwright/tests/login-web/update_password.spec.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {captchaConfigPageTest} from '../../fixtures/captchaConfigPageTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import {clickAndExpectToBeVisible} from '../../utils/clickAndExpectToBeVisible';
+import performLogin, {performLogout} from '../../utils/performLogin';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	captchaConfigPageTest,
+	loginTest()
+);
+
+test.afterEach(
+	'Reset configuration and delete emails',
+	async ({captchaConfigPage}) => {
+		await captchaConfigPage.resetCaptchaConfiguration();
+	}
+);
+
+test('LPD-52234: Check if you can change languages in the update password page', async ({
+	apiHelpers,
+	captchaConfigPage,
+	page,
+}) => {
+	const userAccount = await apiHelpers.headlessAdminUser.postUserAccount();
+
+	try {
+		await captchaConfigPage.disableCaptcha();
+
+		await performLogout(page);
+
+		await page.goto(liferayConfig.environment.baseUrl);
+		await page.getByRole('button', {name: 'Sign In'}).click();
+		await page.getByText('Forgot Password').click();
+		await page.waitForTimeout(1000);
```

Comment:
Instead of waiting for the timeout, can we wait for an element on the Forgot Password page to be present?  It needs to be unique to that page, since Email Address is on both and may return `true` too early.